server/api/products.js

```diff
@@ -0,0 +1,62 @@
+const Sequelize = require('sequelize');
+const router = require('express').Router()
+const { Product } = require('../../db')
+
+router.get('/', async function (req, res, next){
+	try {
+		const products = await Product.findAll()
+  		res.status(200).send(products)
+	} catch (error) {
+		console.log(error)
+		next(error)
+	}
+});
+
+router.get('/:productId', async function (req, res, next){
+	try {
+		const product = await Product.findOne({
+			where: {
+			  id: req.params.productId
+			},
+		})
+  		res.status(200).send(product)
+	} catch (error) {
+		console.log(error)
+		next(error)
+	}  	
+});
+
+router.post('/', function (req, res, next) {
+	Product.create(req.body)
+	.then(product => res.status(201).send(product))
+	.catch(next);
```

Comment:
This is a call where errors are very possible -- the request may not come in with the required fields to create a new Product. There should be more detailed logging and handling of errors here.
For example, consider checking if the required fields are even present in the request body before attempting to create the product, or add default values if they aren't

Reply:
Incorporated with check. Product name is required hence it will not be default updated instead will be thrown back if not found.